{{ cookiecutter.package_name }}/pyproject.toml

```
@@ -5,7 +5,6 @@ requires = [
   "wheel",
 {%- if cookiecutter.use_compiled_extensions == 'y' %}
```

This should look like this, though this change should be made in the openastronomy repo.
```suggestion
{%- if cookiecutter.use_compiled_extensions == 'y' %}
  "extension-helpers",
  "numpy>=1.25",
  "cython",
```